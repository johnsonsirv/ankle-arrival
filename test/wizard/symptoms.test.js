/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import SymptomsPage from '../../src/wizard/symptoms';

function setup() {
  const props = {
    name: 'symptoms',
    onChange: jest.fn(),
    symptoms: [{ id: 5, code: 'a14', description: 'excessive abdominal' }],
  };

  const enzymeWrapper = shallow(<SymptomsPage {...props} />);
  return { props, enzymeWrapper };
}

describe('<SymptomsPage /> rendering', () => {
  const { enzymeWrapper } = setup();
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
