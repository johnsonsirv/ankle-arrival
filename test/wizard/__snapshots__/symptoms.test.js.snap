// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SymptomsPage /> rendering renders correctly 1`] = `ShallowWrapper {}`;
